libs/pages/logs/environment/src/lib/feature/pod-logs-feature/pod-logs-feature.tsx

```diff
@@ -70,7 +70,11 @@ export function PodLogsFeature({ clusterId }: PodLogsFeatureProps) {
     const podFilter = filter.find(({ key }) => key === POD_NAME_KEY)
     return debouncedServiceMessages
       .filter(({ pod_name }) => (podFilter && podFilter.value !== 'ALL' ? podFilter.value === pod_name : true))
-      .concat(enabledNginx ? debouncedInfraMessages.map((message) => ({ ...message, version: '', pod_name: '' })) : [])
+      .concat(
```

Comment:
mmh why ?

Reply:
This anticipates future typescript versions, which will throw errors on this